apps/main/lib/main/views/home.rb

```diff
@@ -0,0 +1,18 @@
+require "main/import"
+require "main/view/controller"
+
+module Main
+  module Views
+    class Home < Main::View::Controller
+      include Main::Import["repositories.articles_repo"]
+
+      configure do |config|
+        config.template = "home"
+      end
+
+      expose :articles do
+        articles_repo.home_page_listing
```

Comment:
asking so I understand the relationship of strings, file names and variable names here
- does `"repositories.articles_repo"` do a `require_relative "../../repositories/articles_repo.rb"`?
- variable `articles_repo` name is extracted from the string? and automagically be assigned with `Main::Repositories::ArticlesRepo` class loaded from the .rb ?